src/playwright-api/makeTest.ts

```diff
@@ -21,9 +22,31 @@ export const makeTest = (
     PlaywrightWorkerArgs & PlaywrightWorkerOptions
   >
 ) =>
-  base.extend<{ save: void }>({
+  base.extend<ChromaticConfig & { save: void }>({
+    // ChromaticConfig defaults
+    delay: [undefined, { option: true }],
+    diffIncludeAntiAliasing: [undefined, { option: true }],
+    diffThreshold: [undefined, { option: true }],
+    disableAutoCapture: [false, { option: true }],
+    forcedColors: [undefined, { option: true }],
+    pauseAnimationAtEnd: [undefined, { option: true }],
+    prefersReducedMotion: [undefined, { option: true }],
```

Comment:
This is interesting. Is this the syntax pattern Playwright requires for options? Just curious, no change needed.

Reply:
Yes, more info here: https://playwright.dev/docs/test-parameterize#parameterized-projects